### ISSUE TITLE
Can only update a mounted or mounting component : Icon component

### ISSUE THREAD

**Issue by pierre-H** [expo/vector-icons#11]
Hello,
I've got a strange error : 
> Warning: Can only update a mounted or mounting comopnent. This usually means you called setState, replaceState, or forceUpdate on an unmounted component. This is a no-op.
>
> Please check the code for the Icon component.

For this code
```js
class Root extends React.Component {
    private label = <View><FontAwesome name="*myIcon*" /> <Text>Hello</Text></View>;

    render() {
        return (
            {this.props.show ? this.label : undefined}
        );
    }
}
```

But this code doesn't emit any warning : 
```js
class Root extends React.Component {
    private label = <View><FontAwesome name="*myIcon*" /> <Text>Hello</Text></View>;

    render() {
        return (
            {label}
        );
    }
}
```
Is it normal ? How can we fix this ?

**Comment by brentvatne** [comment]
Hello! I think I need more context on this -- does the `show` prop change? What's the parent component?

**Comment by amit-saini** [comment]
I am also facing this issue. I did something similar to what @pierre-H did. In my case show prop changes depending on some api response. @brentvatne it will be great if you can help us out in solving this issue.

**Comment by wandri** [comment]
I have the some issue.

Windows 10 - Hyper V, Visual Studio Emulator Android

I have a main render:
```
render() {
  const {navigate} = this.props.navigation;
  return (
    <View style={styles.container} ref="">
      {this.renderImage()}
    </View>
  );
}
```
Which calls:
```
renderImage() {
  if (this.props.hasError) {
    return <p>Sorry! There was an error loading the items</p>;
  }
  if (this.props.isLoading) {
    return <Spinner/>;
  }
  return (
    <View>
      {this.renderScore()}
   ...
    </View>
  )
}
```

which calls:

```
renderScore() {
  return (
    <View style={styles.score}>
      <FontAwesome name={"star-o"} size={35} color={'rgba(255,255,0,1)'}/>
    </View>
  )
}
```

If I replace FontAwesome by:

`<Ionicons  name={"md-start"} size={35} color={'rgba(255,255,0,1)'}/>`

it works.

And I use <FontAwesome name={"star-o"} size={35} color={'rgba(255,255,0,1)'}/> in an another component and it works.

**Comment by brentvatne** [comment]
try out @expo/vector-icons@5.1.0

**Comment by wandri** [comment]
Problem solved with @expo/vector-icons@5.1.0.
Thanks!
